layouts/case-studies/list.html

```diff
@@ -21,6 +21,11 @@
 		</div>
 	</div>
 </section>
+<style>
+    #video button:hover {
+        background-color: #303539;
+	}
+</style>
```

Comment:
Could this change be done by modifying an existing SCSS file instead of directly adding the style in the layout file?

Reply:
sure @dipesh-rawat will take a look into it.

Reply:
hey @dipesh-rawat I am having trouble locating the SCSS file. can you help me? to find the path for the SCSS file

Reply:
Have a look in `assets/scss` - we put the SCSS there.